startup/10-devices.py

```diff
@@ -183,20 +183,25 @@ def _update_readback_setpoint(self):
 
     def put(self, value):
         self._setpoint = float(value)
-
+        
```

Comment:
This addition is not needed. That can be fixed by configuring the pre-commit hook (check how it was done in https://github.com/BNL-ATF/ophyd-basler/blob/main/.github/workflows/pre-commit.yml, and set up as an automatic hook for git: https://nsls-ii.github.io/scientific-python-cookiecutter/pre-commit.html)